File: BEGINNER/Python/1038-Snack.py
X, Y = map(int, input().split())

if(X==1):
    price = 4.0
elif(X==2):
    price = 4.5
elif(X==3):
    price = 5.0
elif(X==4):
    price = 2.0
elif(X==5):
    price = 1.5

total = price * Y

print("Total: R$ %.2f" %total)

File: BEGINNER/PHP/1038-Snack.php
<?php
$itens = array(1 => 4, 2 => 4.5, 3 => 5, 4 => 2, 5 => 1.5);
    
$x = explode(" ", fgets(STDIN));
    
foreach($itens as $code => $price){
    if($x[0] == $code){
        $total = $x[1] * $price;
        echo "Total: R$ " . number_format($total, 2) . "\n";
    }
}
?>
